third-party/cmake/BuildGperf.cmake

```diff
@@ -37,10 +37,15 @@ function(BuildGperf)
 endfunction()
 
 if(UNIX OR (MINGW AND CMAKE_CROSSCOMPILING))
+  if(CMAKE_SYSTEM_NAME MATCHES "OpenBSD")
+    set(MAKE_OpenBSD "MAKE=gmake")
+  else()
+    set(MAKE_OpenBSD "")
+  endif()
 
   BuildGperf(
     CONFIGURE_COMMAND ${DEPS_BUILD_DIR}/src/gperf/configure
-      --prefix=${HOSTDEPS_INSTALL_DIR}
+      --prefix=${HOSTDEPS_INSTALL_DIR} ${MAKE_OpenBSD}
```

Comment:
Does `MAKE=${MAKE_PRG}` work instead? 

`third-party/CMakeLists.txt` already searches for `gmake` and sets `MAKE_PRG` as needed.

Reply:
Yeah you are right! I usually sleep by the time I found this bug (I was already half asleep) 🤪😴